docs/using-direct-csi.md

```diff
@@ -0,0 +1,38 @@
+# Using Direct CSI Driver [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
+
+## Install Direct CSI Driver
+
+### Set the environment variables
+
+```sh
+cat << EOF > default.env
+DIRECT_CSI_DRIVER_PATHS=/var/lib/direct-csi-driver/data{1...4}
+DIRECT_CSI_DRIVER_COMMON_CONTAINER_ROOT=/var/lib/direct-csi-driver
+DIRECT_CSI_DRIVER_COMMON_HOST_ROOT=/var/lib/direct-csi-driver
```

Comment:
What are these environment variables and where are they used?

Reply:
It's for kustomize @kerneltime https://github.com/minio/direct-csi-driver#steps-to-run